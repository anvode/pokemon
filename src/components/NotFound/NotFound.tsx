import * as React from 'react';
import { Link } from 'react-router-dom';

const NotFound: React.FC = () => {

    return (
        <div className="not-found">
            <h1 className="app__title text-center py-3">Page Not Found</h1>
            <div className="jumbotron text-center">
                <Link className="btn btn-primary mt-0" to={'/'}>Back to Pokemon List</Link>
            </div>

        </div>
    );
};

export default NotFound;
