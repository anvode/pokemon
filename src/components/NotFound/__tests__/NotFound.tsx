import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import NotFound from '../NotFound';
import { BrowserRouter } from 'react-router-dom';

afterEach(cleanup);

describe('NotFound', () => {
    it('renders without crashing', () => {
        const { asFragment } = render(<BrowserRouter><NotFound /></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});

