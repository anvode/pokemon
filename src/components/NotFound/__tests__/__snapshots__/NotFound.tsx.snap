// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="not-found"
  >
    <h1
      class="app__title text-center py-3"
    >
      Page Not Found
    </h1>
    <div
      class="jumbotron text-center"
    >
      <a
        class="btn btn-primary mt-0"
        href="/"
      >
        Back to Pokemon List
      </a>
    </div>
  </div>
</DocumentFragment>
`;
